pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -785,3 +803,24 @@ func (c *Cluster) getOSDLabels(osdID int, failureDomainValue string, portable bo
 		portableKey:         strconv.FormatBool(portable),
 	}
 }
+
+//GetVolumeGroupName returns the Volume group name from the given Logical Volume Path
+func GetVolumeGroupName(lvPath string) (string, error) {
```

Comment:
This function does not need to be public? Maybe cleaner to rename it to `getVolumeGroupName()`.

Reply:
Using this function across different packages. `pkg/daemon/ceph/osd/daemon.go`  `pkg/operator/ceph/cluster/osd/spec.go` . So needed to make it public.

Reply:
ok!